File: games.py
n = int(input())
home = []
away = []
count = 0

for i in range(n):
    x, y = input().split(" ")
    home.append(x)
    away.append(y)

for i in home:
    if i in away:
        count += away.count(i)

print(count)

File: games.java
import java.util.Scanner;

public class games{

	static Scanner in = new Scanner(System.in);

	public static void main(String[]args){
	
		int n = in.nextInt();
		int ans = 0;
		int [] home = new int[n];
		int [] away = new int[n];
		
		for(int i = 0; i < n; i++){
			home[i] = in.nextInt();
			away[i] = in.nextInt();
		
		}

		for(int i = 0; i < n; i++){
			for(int j = 0; j < n; j++){

				if(home[i] == away[j]){
					ans++;
				}
			}
		}

		System.out.println(ans);
		
		}
	}
